docs/intro/install.md

```diff
@@ -50,5 +50,9 @@ guide](../community/developers.md).
 
 ## Next Steps
 
-After installing Krustlet, you can go through [the tutorial](tutorial01.md) to learn how to start
-using Krustlet.
+After installing Krustlet, if you'd like to get started and see something running, go checkout any
+one of the [demos](../../demos). Each of them has a prebuilt WASM module stored in a registry and a
```

Comment:
nit: WASM module is a bit like calling an ATM an "ATM machine". :) "module" is part of the acronym.

What about calling it a "WebAssembly module" instead?

Reply:
I stand corrected... WASM is short for WebAssembly.

Reply:
But but but...my acronyms

Reply:
I think I'll change it just for clarity, but this is definitely not the only acronym used this way (where you say part of what is in the acronym after using it)